avaframe/ana1Tests/damBreak.py

```diff
@@ -193,67 +160,304 @@ def damBreakSol(avaDir, cfg, cfgC):
             main avaframe configuration - here showPlot flag used
         cfgC: configParser object
             configuration setting for avalanche simulation including DAMBREAK section
+        outDirTest: pathlib path
+            path to output directory
 
         Returns
         -------
-        hL: float
-            release thickness
-        h: numpy array
-            flow depth
-        u: numpy array
-            flow velocity
-        phi: float
-            slope angle
-        x: numpy array
-            extent of horizontal domain
+        solDam: dict
+            analytic solution dictionary:
+                tAna: 1D numpy array
+                    time array
+                h0: float
+                    release thickness
+                hAna: 2D numpy array
+                    Flow thickness (rows for x and columns for time)
+                uAna: 2D numpy array
+                    flow velocity (rows for x and columns for time)
+                xAna: 2D numpy array
+                    extent of domain in the horizontal plane coordinate system (rows for x and columns for time)
+                xMidAna: 1D numpy array
+                    middle of the material in x dir in the horizontal plane coordinate system
+                    (used to compute the error)
     """
 
     # Set Parameters
     # Coordinate system chosen in the direction of the inclined plane
     g = cfgC['GENERAL'].getfloat('gravAcc')       # acceleration due to gravity [ms-2]
-    phi = np.radians(22)                          # slope angle [°]
-    delta = np.radians(21)                        # bed friction angle [°]
+    phiDeg = cfgC['DAMBREAK'].getfloat('phi')       # acceleration due to gravity [ms-2]
+    deltaDeg = cfgC['DAMBREAK'].getfloat('delta')       # acceleration due to gravity [ms-2]
```

Comment:
slople angle instead of acceleration...
bed friction... instead of acceleration...